### PR TITLE
Include a note for kernel[-rt] CVEs

### PR DESCRIPTION
**Pull request by vfreex**
So that QE and other people will know that kernel CVEs are cloned as non-CVE bugs on purpose.

### CODE REVIEW

**Review by joepvd — APPROVED** [review]
/lgtm
/approve

File: elliott/elliottlib/cli/find_bugs_kernel_cli.py [hunk 1]
```diff
@@ -253,26 +253,30 @@ def _new_jira_fields_from_bug(bug: Bug, ocp_target_version: str, kmaint_tracker:
...
-        #         cve_flaws.append(flaw_bug)
-        #     labels = {"Security", "SecurityTracking"}
-        #     cve_names = re.findall(r"(CVE-\d+-\d+)", bug.summary)
-        #     labels |= set(cve_names)
-        #     labels |= {f"pscomponent:{component}" for component in bug.components}
-        #     labels |= {f"flaw:bz#{flaw.id}" for flaw in cve_flaws}
-        #     fields["labels"] += sorted(labels)
+            hint += "Please note that this bug is cloned as a non-CVE bug intentionally due to limitations in interoperability in internal processes.\n"
+
+        fields["description"] = f"{hint}\n----\n{bug.description}"
```
Comment (Ximinhan) [inline elliott/elliottlib/cli/find_bugs_kernel_cli.py:279]: hint already have `\n`

Comment (vfreex) [reply elliott/elliottlib/cli/find_bugs_kernel_cli.py:279]: It is fine. I intentionally added an extra \n to make a clear seperation.

**Pull request merged**